python/pyhdk/_common.pyx

```diff
@@ -7,10 +7,10 @@ from libcpp.memory cimport unique_ptr, make_unique, shared_ptr
 from cython.operator cimport dereference
 
 cdef class SQLType:
```

Comment:
It would be nice to also rename this one to `TypeId` similarly to `SQLTypeInfo`. I think we should avoid the impression that HDK is an SQL engine.